dso-l2/src/main/java/com/tc/config/schema/setup/StandardConfigurationSetupManagerFactory.java

```diff
@@ -157,6 +159,10 @@ public static Options createOptions(ConfigMode configMode) {
       l2NameOption.setRequired(false);
       l2NameOption.setArgName("l2-name");
       options.addOption(l2NameOption);
+
+      Option safeModeOption = new Option("s", "safe", false, "to start the server in safe mode");
```

Comment:
This looks like a new consistency mode.  Availability with safe startup.  It really should be a stripe wide configuration.  It doesn't make much sense to have a single node in this mode and all the others not in this mode.  We can improve this with later, what do you think?

Reply:
This option is to be used only on cluster restarts. For the initial start of the cluster this option doesn't make much sense. That's why I kept it as an option. Even on restart it's upto the customer whether to start in this safe mode or the regular high availability mode. So I feel this being an option gives more flexibility.

Reply:
After thinking a bit more, having a dedicated consistency mode would be a good idea so that users won't have to worry about providing extra options on startups and even across all servers in all stripes. That's more error prone although it provides better flexibility. We'll definitely discuss and improve this later.